cmd/kots/cli/install.go

```diff
@@ -371,6 +383,25 @@ func InstallCmd() *cobra.Command {
 				}
 			}()
 
+			authSlug, err := auth.GetOrCreateAuthSlug(clientset, deployOptions.Namespace)
+			if err != nil {
+				return errors.Wrap(err, "failed to get kotsadm auth slug")
+			}
+
+			if deployOptions.License != nil {
+				if err := ValidateAutomatedInstall(deployOptions, authSlug, apiEndpoint); err != nil {
```

Comment:
the task status is not being reset at the end, so if the user retries a failed installation for example, there could be a race condition where the old status is read and the install command will fail instead of waiting for the new status.